test/algodclientv2_test.go

```diff
@@ -5,7 +5,6 @@ import (
 	"context"
```

Comment:
These changes are technically unrelated to code generation.

The first version of our unit tests took a specific response and verified a single field. This replaces the single-field checks with the JSON compare functions.